stacks-in-depth/nodes-and-miners/mine-mainnet-stacks-tokens.md

```diff
@@ -90,26 +90,58 @@ After this runs, you should see some JSON printed to the screen that looks like
 
 ```json
 {
-  "mnemonic": "exhaust spin topic distance hole december impulse gate century absent breeze ostrich armed clerk oak peace want scrap auction sniff cradle siren blur blur",
+  "mnemonic": "stomach strong dog warfare drip garbage vendor among argue flash anchor change any credit purity learn gas unfold urban electric van alcohol lobster reflect",
   "keyInfo": {
-    "privateKey": "2033269b55026ff2eddaf06d2e56938f7fd8e9d697af8fe0f857bb5962894d5801",
-    "address": "STTX57EGWW058FZ6WG3WS2YRBQ8HDFGBKEFBNXTF",
-    "btcAddress": "mkRYR7KkPB1wjxNjVz3HByqAvVz8c4B6ND",
+    "privateKey": "4b48f79eea88bb20de75148ecc8e5480463048b937d9c77801b388b4e1b49d2d01",
+    "publicKey": "03a707458e468f7ee6513f8135500780576dca2b52c98ae909a538f75e4901aad5",
+    "address": "SP2D9W82GHM1V0ACDRVCDNKJV0J6NS7KJ25WHHRMH",
+    "btcAddress": "1F6YVM1E633QDUo9zgT3q7K3T9FYuKJ3RZ",
+    "wif": "Kyk49jsPGen5C1ThhyJJH4CndLk8yLESuQJVGsbbTV3FFF9CRTJG",
```

Comment:
I would prefer to use the same fake values in the miner-docs repo here. `1F6YVM1E633QDUo9zgT3q7K3T9FYuKJ3RZ` is a valid address

Reply:
The old `make_keychain` didn't have a `wif` field, so i updated them with the new ones. Should i change the generated keychain back to the old one and just add a random wif?

Reply:
intersting. it should have since it's part of the output. 
i would use the psuedo values from the miner-docs repo here for both mainnet and testnet sections

Reply:
Got it, will do that, thank you!